config/samples/custom-role.yaml

```diff
@@ -0,0 +1,117 @@
+# (c) Copyright [2021-2022] Micro Focus or one of its affiliates.
```

Comment:
Do we need to provide 2 roles?  One for the manager and one for the leader election?  Or is the leader election role a subset of the manager one?

Reply:
Yes this file combines both manager and leader election rules. Do you think it is better to keep them separate?

Reply:
I don't mind having one.